### ISSUE TITLE
Contact grid button text shows through slide over

### ISSUE THREAD

**Issue by bradrf** [tailwindlabs/tailwindui-issues#493]
**What component (if applicable)**
- URL for category: 
   - https://tailwindui.com/components/application-ui/lists/grid-lists
   - https://tailwindui.com/components/application-ui/overlays/slide-overs
- Component name: 
   - Contact cards with small portraits
   - Create project form example

**Describe the bug**
When the slide over is shown on top of the contact cards, parts of the cards are visible within the slide over content area (e.g. the button icons and text).

**To Reproduce**
1. Create a base page with the contact card layout
2. Add a button to show the slide over on top of the contact cards
4. Note the button contact info on the far right will "bleed" through the slide over content

**Expected behavior**
The background for the slide over would have a large z-index and be opaque for any elements underneath.

**Screenshots**
![image](https://user-images.githubusercontent.com/295715/107800886-391e6980-6d14-11eb-9bad-46656801afb9.png)

**Browser/Device (if applicable)**
 - OS: Linux (Pop!_OS 20.10)
 - Browser: Chrome 
 - Version: 88.0.4324.150

**Comment by davidluhr** [comment]
@bradrf Using Tailwind Play, I don't seem to be able to recreate this issue, even without adding a `z-index` class to the Slide Over's container element: https://play.tailwindcss.com/EwbLa0JyhH

Do your slideover come after the Grid Lists in the DOM? The Grid List component doesn't use `z-` classes, so it's odd for that text to be sitting above the Slide Over.

Thanks for your help in troubleshooting this!

**Comment by bradrf** [comment]
First, thank you for pointing out Tailwind Play! I didn't think to look for that as a resource. Very handy!

I found the "problem", though I'll let you tell me if it's just to be expected (I'm not a strong CSS or Tailwind developer). If you swap the order of those by placing the Slide Over _before_ the other content, it will exhibit the "bleed through" behavior. I can definitely  swap the order on my project, of course.

Feel free to close if you think this should be expected.

**Comment by davidluhr** [comment]
@bradrf Ahh that makes sense. The stacking context of the browser means that elements further down in the DOM render on top by default, unless previous elements have a sufficiently high `z-index`. Closing this for now, as you've found a solution, and this is expected browser behavior. But, know that we've made a note in the team to think through a global z-index system to help prevent situations like this, regardless of how the components are used.
